mysite/templates/main_index.html

```diff
@@ -214,14 +214,16 @@ <h4 class="modal-title" id="ad-title">Calcont Whiteboard</h4>
     </script>
     <script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-3911342542125154" crossorigin="anonymous"></script>
     <!-- recommend-calcont-adunit -->
-    <ins class="adsbygoogle" style="display:block" data-ad-client="ca-pub-3911342542125154" data-ad-slot="3259871936" data-ad-format="auto" data-full-width-responsive="true"></ins>
-    <script>
-      ;(adsbygoogle = window.adsbygoogle || []).push({})
-    </script>
+    <div class="add-container">
```

Comment:
I don't think so this is google ads issue .
![image](https://github.com/calcont/calcont.in/assets/71604396/79591fcb-685f-4010-8364-7a0eceb3c6ff)


Can u verify from ur end as well ?